src/README.md

```diff
@@ -32,5 +32,7 @@ Developer guides contain examples of code, mostly written in JavaScript and Soli
 ### Other resources
 
 * JavaScript [@unstoppable/resolution](https://www.npmjs.com/package/@unstoppabledomains/resolution) library \[[github](https://github.com/unstoppabledomains/resolution)\]
+* Java [Maven Central](https://search.maven.org/artifact/com.unstoppabledomains.resolution/resolution/1.1.0/jar) library \[[github](https://github.com/unstoppabledomains/resolution-java)\]
+* Swift [Cocoapods](https://cocoapods.org/pods/UnstoppableDomainsResolution) library \[[github](https://github.com/unstoppabledomains/resolution-swift)\]
```

Comment:
I'd prefer to keep a full library name, instead of "Maven Central" or "Cocoapods"

Reply:
Like resolution-java nd resolution-swift?

Reply:
Say, "com.unstoppabledomains.resolution" for Java and "UnstoppableDomainsResolution" for Swift. It should be consistent: for npm specify the package name. Btw, I've just realized it's "@unstoppable/resolution", instead of "@unstoppabledomains/resolution"... :) While we are here, could you fix it as well?

Reply:
Thanks!